src/Libraries/CoreNodeModels/Input/IntegerSlider.cs

```diff
@@ -33,14 +33,15 @@ public override string NodeType
         {
             get
             {
-                return "IntegerRangeInputNode";
+                return "NumberInputNode";
             }
         }
 
         [JsonConstructor]
         private IntegerSlider(IEnumerable<PortModel> inPorts,
             IEnumerable<PortModel> outPorts): base(inPorts, outPorts)
         {
+            NumberType = "Integer";
             Min = 0;
```

Comment:
have you verified that if you change these values and deserialize they are actually set correctly and not to these defaults?